docs/guides/development/versions.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Versions & releases
+---
+
+# Managing versions and releases
+
+Building a Pack is often an incremental process, and Coda provides a version and release mechanic to make it possible to add new features or fix bugs in a way that doesn't impact existing users.
+
+## Versions
+
+Versions are like checkpoints for your code, and represent a unique build of your Pack that could be run in a live doc. Versions and version numbers are private to the Pack maker, and not visible to other users that install your Pack. If you are familiar with [Git][git], they are sort of like a Git commit.
+
+A new version is created when:
+
+- You press the **Build** button in the Pack Studio web editor, and the build completes successfully.
+- You run `npx coda upload` using the Packs CLI.
+
+!!! info "Many Pack versions"
+    When working in the Pack Studio web editor the only way to test your code is to build it and try it in a live doc, therefore it's not uncommon to accumulate many Pack versions. While this may feel messy, rest assured it has no impact on how your Pack looks or works.
+
+
+### Version numbers
+
+By default version numbers are automatically generated for you, as a set of incrementing whole numbers (1, 2, 3, etc). In various places throughout the user interface they are presented with a preceding "v" character, to indicate they are a version number ("v1", "v2", etc).
+
+It is possible to manually set the version numbers, by calling the `setVersion()` method in your code:
+
+```ts
+export const pack = coda.newPack();
+pack.setVersion("1.0.5");
+```
+
+There are some restrictions to the version numbers you can select however:
+
+- The version numbers must increase with each new build / upload.
+- The version numbers must follow the [SemVer standard][semver].
```

Comment:
Might be worth adding a simple example so people don't have to grok the spec. Can be as simple as "That is, versions must be of the form `1`, `1.2`, or `1.2.3`."

Reply:
Done.